docker_hub/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:latest
+ARG ADAPT_VERSION
+RUN apk --update add --no-cache nodejs npm openssh \
+  && rm -rf /var/cache/apk/*
```

Comment:
You shouldn't need to remove /var/cache/apk. The `--no-cache` option should take care of that for you.

Reply:
Done and confirmed it works as expected.